server/application/rest/service_shifts.py

```diff
@@ -40,6 +78,15 @@ def handle_service_shift():
             else None
         )
 
+        # Authorization check for viewing service shifts for a specific shelter
+        if not is_authorized(repo, user_email, Resources.SHIFT, shelter_id):
+            return Response(
+                json.dumps({"message": "Unauthorized to view service "
+                "shifts for this shelter"}),
+                mimetype="application/json",
+                status=HTTP_STATUS_CODES_MAPPING[ResponseTypes.FORBIDDEN]
+            )
+
```

Comment:
```suggestion
```
Anyone who is authenticated should be allowed to view all defined shifts. Volunteer dashboard will use the API endpoint to show volunteers which shifts can be signed up for.